spec-tests/src/test/java/org/candlepin/spec/content/OwnerContentSpecTest.java

```diff
@@ -1157,4 +957,1009 @@ public void shouldPermitSafeParallelModification() throws Exception {
+            OwnerDTO owner = this.owners.get(1);
+
+            List<String> expectedPids = this.productMap.values()
+                .stream()
+                .filter(this.buildOwnerProductPredicate(owner))
+                .map(ProductInfo::id)
+                .toList();
+
+            List<String> expectedCids = getContentIdsFromPids(expectedPids);
+
+            List<ContentDTO> output = this.adminClient.ownerContent()
+                .getContentsByOwner(owner.getKey(), null, null, INCLUSION_INCLUDE, INCLUSION_INCLUDE);
+
+            assertThat(output)
+                .isNotNull()
+                .map(ContentDTO::getId)
+                .containsAll(expectedCids);
+        }
+
+        @Test
+        public void shouldAllowFilteringWithCustomExcluded() {
+            OwnerDTO owner = this.owners.get(1);
+
+            List<String> expectedPids = this.productMap.values()
+                .stream()
+                .filter(this.buildOwnerProductPredicate(owner))
+                .filter(pinfo -> !pinfo.custom())
+                .map(ProductInfo::id)
+                .toList();
+
+            List<String> expectedCids = getContentIdsFromPids(expectedPids);
+
+            List<ContentDTO> output = this.adminClient.ownerContent()
+                .getContentsByOwner(owner.getKey(), null, null, INCLUSION_INCLUDE, INCLUSION_EXCLUDE);
+
+            assertThat(output)
+                .isNotNull()
+                .map(ContentDTO::getId)
+                .containsAll(expectedCids)
+                .doesNotContainAnyElementsOf(this.getUnexpectedIds(expectedPids));
+        }
+
+        @Test
+        public void shouldAllowFilteringWithCustomExclusive() {
+            OwnerDTO owner = this.owners.get(1);
+
+            List<String> expectedPids = this.productMap.values()
+                .stream()
+                .filter(ProductInfo::custom)
+                .filter(pinfo -> pinfo.owner().getKey().equals(owner.getKey()))
+                .map(ProductInfo::id)
+                .toList();
+
+            List<String> expectedCids = getContentIdsFromPids(expectedPids);
+
+            List<ContentDTO> output = this.adminClient.ownerContent()
+                .getContentsByOwner(owner.getKey(), null, null, INCLUSION_INCLUDE, INCLUSION_EXCLUSIVE);
+
+            assertThat(output)
+                .isNotNull()
+                .map(ContentDTO::getId)
+                .containsAll(expectedCids)
+                .doesNotContainAnyElementsOf(this.getUnexpectedIds(expectedPids));
+        }
+
+        @Test
+        public void shouldErrorWithInvalidCustomInclusion() {
+            OwnerDTO owner = this.owners.get(1);
+
+            assertBadRequest(() -> this.adminClient.ownerContent()
+                .getContentsByOwner(owner.getKey(), null, null, INCLUSION_INCLUDE, "invalid_type"));
+        }
+
+        @Test
+        public void shouldAllowQueryingWithMultipleFilters() {
+            Random rand = new Random(14023);
```

Comment:
As a note: you'll want to check the data selected with each of these seeds to make sure it's somewhat representative of the shape we're looking for. The test should still work regardless, but it's nice when it gives us distributed data.

Reply:
Updated

Reply:
I see what you mean. I updated this seed to get more than one content ID. I was having trouble finding a value to get more than two.

[o2-content-1a, o2-content-1b]